Cognite.Extensions/CogniteResult.cs

```diff
@@ -752,7 +799,15 @@ public enum ResourceType
         /// <summary>
         /// None or unknown
         /// </summary>
-        None = -1
+        None = -1,
+        /// <summary>
+        /// Data modeling space id
+        /// </summary>
+        SpaceId,
+        /// <summary>
+        /// Data modeling instance id
+        /// </summary>
+        InstanceId
```

Comment:
Move these above None, just prefer to have that at the bottom still.

Reply:
The reason I put them below None is because C# enums are assigned numerical values based on their order when not explicitly assigned a numerical value, so it is breaking to change the order.

Reply:
Yes, but `None` _has_ a numerical value.

Reply:
Ah, true, I will pop those up